src/main/resources/workflows/0.1.0/irida_workflow.xml

```diff
@@ -40,46 +40,46 @@
-      <toolParameter toolId="toolshed.g2.bx.psu.edu/repos/nml/staramr/staramr_search/0.7.0" parameterName="advanced.minimum_contig_length"/>
+      <toolParameter toolId="toolshed.g2.bx.psu.edu/repos/nml/staramr/staramr_search/0.7.1" parameterName="advanced.minimum_contig_length"/>
     </parameter>
     <parameter name="staramr-2-advanced.minimum_N50_value" defaultValue="10000">
-      <toolParameter toolId="toolshed.g2.bx.psu.edu/repos/nml/staramr/staramr_search/0.7.0" parameterName="advanced.minimum_N50_value"/>
+      <toolParameter toolId="toolshed.g2.bx.psu.edu/repos/nml/staramr/staramr_search/0.7.1" parameterName="advanced.minimum_N50_value"/>
     </parameter>
     <parameter name="staramr-2-advanced.genome_size_upper_bound" defaultValue="6000000">
-      <toolParameter toolId="toolshed.g2.bx.psu.edu/repos/nml/staramr/staramr_search/0.7.0" parameterName="advanced.genome_size_upper_bound"/>
+      <toolParameter toolId="toolshed.g2.bx.psu.edu/repos/nml/staramr/staramr_search/0.7.1" parameterName="advanced.genome_size_upper_bound"/>
     </parameter>
     <parameter name="staramr-2-advanced.exclude_resistance_phenotypes" defaultValue="false">
-      <toolParameter toolId="toolshed.g2.bx.psu.edu/repos/nml/staramr/staramr_search/0.7.0" parameterName="advanced.exclude_resistance_phenotypes"/>
+      <toolParameter toolId="toolshed.g2.bx.psu.edu/repos/nml/staramr/staramr_search/0.7.1" parameterName="advanced.exclude_resistance_phenotypes"/>
     </parameter>
     <parameter name="staramr-2-advanced.plasmidfinder_type" defaultValue="include_all">
-      <toolParameter toolId="toolshed.g2.bx.psu.edu/repos/nml/staramr/staramr_search/0.7.0" parameterName="advanced.plasmidfinder_type"/>
+      <toolParameter toolId="toolshed.g2.bx.psu.edu/repos/nml/staramr/staramr_search/0.7.1" parameterName="advanced.plasmidfinder_type"/>
     </parameter>
     <parameter name="staramr-2-advanced.report_all_blast" defaultValue="false">
-      <toolParameter toolId="toolshed.g2.bx.psu.edu/repos/nml/staramr/staramr_search/0.7.0" parameterName="advanced.report_all_blast"/>
+      <toolParameter toolId="toolshed.g2.bx.psu.edu/repos/nml/staramr/staramr_search/0.7.1" parameterName="advanced.report_all_blast"/>
     </parameter>
     <parameter name="staramr-2-advanced.unacceptable_number_contigs" defaultValue="1000">
-      <toolParameter toolId="toolshed.g2.bx.psu.edu/repos/nml/staramr/staramr_search/0.7.0" parameterName="advanced.unacceptable_number_contigs"/>
+      <toolParameter toolId="toolshed.g2.bx.psu.edu/repos/nml/staramr/staramr_search/0.7.1" parameterName="advanced.unacceptable_number_contigs"/>
     </parameter>
     <parameter name="staramr-2-advanced.pid_threshold" defaultValue="98.0">
-      <toolParameter toolId="toolshed.g2.bx.psu.edu/repos/nml/staramr/staramr_search/0.7.0" parameterName="advanced.pid_threshold"/>
+      <toolParameter toolId="toolshed.g2.bx.psu.edu/repos/nml/staramr/staramr_search/0.7.1" parameterName="advanced.pid_threshold"/>
```

Comment:
In this file could you make sure to update the `repository` entry as well with the new tool id.